components/script/dom/performance.rs

```diff
@@ -120,7 +120,8 @@ pub struct Performance {
 impl Performance {
     fn new_inherited(global: &GlobalScope,
                      navigation_start: u64,
-                     navigation_start_precise: f64) -> Performance {
+                     navigation_start_precise: u64) -> Performance {
+        let navigation_start_precise = navigation_start_precise as f64;
```

Comment:
Let's change PerformanceTiming to accept a u64 as well, and make PerformanceTiming::navigation_start_precise() return u64. We can simplify Performace::now() to only cast the result of the calculation instead.